common/configuration/projectconfig_resolver_test.go

```diff
@@ -40,6 +40,37 @@ func TestSet(t *testing.T) {
 	})
 }
 
+func TestGetGlobal(t *testing.T) {
```

Comment:
This test is not actually related to my change. I just added it to give myself some extra confidence that I was building the right thing (i.e. the fallback-to-global behavior) and then decided to keep this in for some extra branch coverage.